tests/xbot/common/subsystems/MockPoseSubsystem.java

```diff
@@ -0,0 +1,34 @@
+package xbot.common.subsystems;
+
+import com.google.inject.Inject;
+
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.XPropertyManager;
+
+public class MockPoseSubsystem extends AbstractPoseSubsystem {
+
+    private XCANTalon left;
```

Comment:
Ignore-able suggestion, but since this Mock class is just about getting/setting the left/right distance, maybe just have methods that do that instead of needing to involve XCANTalon at all?